tests/manager/search/test_external_search.py

```diff
@@ -131,6 +122,85 @@ def query_works_multi(self, queries, debug=False):
         assert pagination.offset == default.offset
         assert pagination.size == default.size
 
+    def test_remove_work(
+        self,
+        end_to_end_search_fixture: EndToEndSearchFixture,
+        db: DatabaseTransactionFixture,
+        caplog: pytest.LogCaptureFixture,
+    ):
+        moby_duck = db.work(title="Moby Duck", with_open_access_download=True)
+        moby_dick = db.work(title="Moby Dick", with_open_access_download=True)
```

Comment:
I had to look at this a couple of times because I didn't catch the difference. Moby Duck. 🦆

Reply:
I think I pulled this from an older test. It is hard to see the diff though. I renamed things a bit to make it easier to see whats happening at a glance.